thirdparty/fluidsynth/fluidsynth-2.1.4/src/utils/fluidsynth_priv.h

```diff
@@ -244,21 +244,21 @@ void* fluid_alloc(size_t len);
 #define FLUID_SPRINTF                sprintf
 #define FLUID_FPRINTF                fprintf
 
-#if (defined(WIN32) && _MSC_VER < 1900) || defined(MINGW32)
-/* need to make sure we use a C99 compliant implementation of (v)snprintf(),
- * i.e. not microsofts non compliant extension _snprintf() as it doesn't
- * reliably null-terminate the buffer
- */
-#define FLUID_SNPRINTF           g_snprintf
-#else
+//#if (defined(WIN32) && _MSC_VER < 1900) || defined(MINGW32)
```

Comment:
Sure that won't break MinGW 32 builds?

Reply:
Yes, I checked

Reply:
what is the current `_MSC_VER`?

Reply:
1926

Reply:
So that change should not be relevant at all it seems, we ended up in the else branch anyhow (except for MinWG 32)?

Reply:
Yes, this is to fix the MinGW build

Reply:
Hmm, then just remove that part rather than breaking old MSVC builds?

Reply:
Then again I don't recall this ever having been a problem with MinGW

Reply:
I removed the use of `glib` so that there would be no dependence on it, so that it would be easier for us to build the application.
Therefore, in any case, we will not have a build that uses functions from `glib`, for example, `g_snprintf`. So I commented out this code completely.
Define MINGW32 is set for any build with MINGW, including 64-bit (probably for historical reasons)
To get this problem, need to build the current master with the `BUILD_MU4_UI` option with MinGW (any, 32 or 64)

Reply:
Ah, I see, thanks